src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.DateTime.cs

```diff
@@ -0,0 +1,284 @@
+        /// </remarks>
+        public void WriteString(string propertyName, DateTime value, bool escape = true)
+            => WriteString(propertyName.AsSpan(), value, escape);
+
+        /// <summary>
+        /// Writes the property name and <see cref="DateTime"/> value (as a JSON string) as part of a name/value pair of a JSON object.
+        /// </summary>
+        /// <param name="propertyName">The UTF-16 encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        /// <param name="value">The value to be written as a JSON string as part of the name/value pair.</param>
+        /// <param name="escape">If this is set to false, the writer assumes the property name is properly escaped and skips the escaping step.</param>
+        /// <exception cref="ArgumentException">
+        /// Thrown when the specified property name is too large.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
+        /// </exception>
+        /// <remarks>
+        /// Writes the <see cref="DateTime"/> using the round-trippable 'O' <see cref="StandardFormat"/> , for example: 2017-06-12T05:30:45.7680000.
+        /// </remarks>
+        public void WriteString(ReadOnlySpan<char> propertyName, DateTime value, bool escape = true)
+        {
+            JsonWriterHelper.ValidateProperty(propertyName);
+
+            if (escape)
+            {
+                WriteStringEscape(propertyName, value);
+            }
+            else
+            {
+                WriteStringByOptions(propertyName, value);
+            }
+
+            SetFlagToAddListSeparatorBeforeNextItem();
+            _tokenType = JsonTokenType.String;
+        }
+
+        /// <summary>
+        /// Writes the property name and <see cref="DateTime"/> value (as a JSON string) as part of a name/value pair of a JSON object.
+        /// </summary>
+        /// <param name="propertyName">The UTF-8 encoded property name of the JSON object to be written.</param>
+        /// <param name="value">The value to be written as a JSON string as part of the name/value pair.</param>
+        /// <param name="escape">If this is set to false, the writer assumes the property name is properly escaped and skips the escaping step.</param>
+        /// <exception cref="ArgumentException">
+        /// Thrown when the specified property name is too large.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
+        /// </exception>
+        /// <remarks>
+        /// Writes the <see cref="DateTime"/> using the round-trippable 'O' <see cref="StandardFormat"/> , for example: 2017-06-12T05:30:45.7680000.
```

Comment:
It's probably more correct to say "Writes the DateTime using the round-trippable ('O') StandardFormat", with the 'O' in parens; but that's really a doc writer thing, so not very important for this PR.